zipkin-storage/elasticsearch/src/main/java/zipkin/storage/elasticsearch/InternalElasticsearchClient.java

```diff
@@ -49,7 +49,7 @@
    *
    * <p> See https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-from-size.html
    */
-  protected static final int MAX_RAW_SPANS = 10000; // the default elasticsearch allowed limit
+  protected static final int MAX_RAW_ITEMS = 10000; // the default elasticsearch allowed limit
```

Comment:
maybe `MAX_RESULT_WINDOW` instead, as that matches `index.max_result_window` which is what this is about?